app/lib/champaign_queue/clients/sqs.rb

```diff
@@ -6,22 +6,24 @@ class Sqs
       class << self
         # +params+ - The message to send. String maximum 256 KB in size.
         # +delay+  - The number of seconds (0 to 900 - 15 minutes) to delay a specific message.
-        def push(params, delay: 0)
-          new(params, delay).push
+        def push(params, group_id:, delay: 0)
+          new(params, group_id, delay).push
```

Comment:
Yeah, I agree the serialization logic should be extracted. We're already doing it for actions and calls, but most of the other events are formatted and pushed in-line from wherever they're needed.